spec/lib/pdf_filler/nc_d400_pdf_spec.rb

```diff
@@ -29,6 +30,10 @@
                primary_esigned_at: tomorrow_midnight)
       }
 
+      before do
+        intake.direct_file_data.mailing_apartment = mailing_apartment
+      end
```

Comment:
[dust] I won't block on this, but why was this added? is it related to changing the state_file_nc_intake factory? it makes it seem like there is something special related to the mailing_apartment in this test, which there shouldn't be. 

also, the test shows `expect(pdf_fields['y_d400wf_apartment number']).to eq 'Apt 2'` - why not 'Apt 2B'?

Reply:
hmm yeah i can go either way here -- I guess I was worried I was losing the apartment number part of the test as I changed the fixture used in this story. `nc_nick` had a apartment address but `nc_daffy_single` doesn't. Maybe I should break this out into its own context if I want to make sure to test it?

Reply:
@mpidcock i think you uncovered a bug here, I wasn't thinking when I made these tests but looks like NCD400 Apartment number has a limit of 5 characters, so if we try to set it to "Apt 2B" it only is able to set it to "Apt 2". Not sure how important it is but since the PDF looks like it has "Apartment Number" over the input field, wondering if we should try to remove any `COMMON_ADDRESS_ABBREV`?

Reply:
For this test I'll update the test to fit the box, so it doesn't lead to confusion